src/Bot.Discord/Commands/InviteCommands.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Bot.Discord.Components;
+using Color_Chan.Discord.Commands.Attributes;
+using Color_Chan.Discord.Commands.Attributes.ProvidedRequirements;
+using Color_Chan.Discord.Commands.MessageBuilders;
+using Color_Chan.Discord.Commands.Modules;
+using Color_Chan.Discord.Core.Common.Models.Embed;
+using Color_Chan.Discord.Core.Common.Models.Interaction;
+using Color_Chan.Discord.Core.Results;
+
+namespace Bot.Discord.Commands;
+
+/// <summary>
+///     The command module for inviting the bot to other Discord's servers.
+/// </summary>
+[UserRateLimit(5, 10)] // Sets the rate lcimit for this command module to 5 requests per 10 seconds per user.
+public class InviteCommands : SlashCommandModule
+{
+    public const string InviteCommandName = "invite";
```

Comment:
These will need to be added to the help command as well. I think page 3 will be good :)

Reply:
Sure, it was my first approach